config.toml

```diff
@@ -1,19 +1,13 @@
-# filename: config.toml
+[db]
+  DataDir = "/home/d/.gossamer"
```

Comment:
Still unable to set DataDir from `config.toml` seems to always use default?

I don't see where any changes were made to affect that in this PR though?

Reply:
I have fixed that in #349, please see my latest comment there